engines/wintermute/detection_tables.h

```diff
@@ -284,6 +287,32 @@ static const WMEGameDescription gameDescriptions[] = {
 	// Escape from the Mansion
 	WME_WINENTRY("escapemansion", "1.3",
 		WME_ENTRY1s("data.dcp",  "1e5d231b56c8a228cd15cb690f50253e", 29261972), Common::EN_ANY, ADGF_UNSTABLE, LATEST_VERSION),
+#ifdef ENABLE_FOXTAIL
+	// FoxTail 1.2.230.1291 (English)
+	WME_WINENTRY("foxtail", "1.2.230.1291",
+		WME_ENTRY1s("data.dcp", "651ae5b062073021edaca7e1de131eec", 59357572), Common::EN_ANY, ADGF_UNSTABLE | ADGF_DEMO, LATEST_VERSION),
```

Comment:
why is it marked as ADGF_DEMO?

Reply:
Currently there are 2 chapters avaliable.
Technically, this still is a "Game in development", but in fact this game is bigger then some finished Wintermute games.
So I'm not sure if it should be ADGF_DEMO...